src/Symfony/Component/Validator/Tests/Validator/AbstractValidatorTest.php

```diff
@@ -548,8 +554,12 @@ public function testArrayReference()
         $this->assertNull($violations[0]->getCode());
     }
 
-    // https://github.com/symfony/symfony/issues/6246
-    public function testRecursiveArrayReference()
+    /**
+     * https://github.com/symfony/symfony/issues/6246.
```

Comment:
let's remove this, we don't keep references to the repo in the code

Reply:
done. there are others but removing them would not be related to this PR really.